src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -515,38 +516,51 @@ const NetworkModificationNodeEditor = ({ currentTreeNode }) => {
 
     const renderNetworkModificationsListTitleLoading = () => {
         return (
-            <Typography className={classes.modificationsTitle}>
-                <CircularProgress
-                    size={'1em'}
-                    className={classes.circularProgress}
-                />
-                <FormattedMessage id={messageId} />
-            </Typography>
+            <div className={classes.modificationsTitle}>
+                <div style={{ width: '5%' }}>
```

Comment:
Create a class to avoid repeating 5% 3 times
Also, maybe we should use theme spacing 1 or 2 instead ?